superset/connectors/sqla/views.py

```diff
@@ -411,6 +409,36 @@ def _delete(self, pk):
         db.session.commit()
         self.update_redirect()
 
+    def get_deployment_names(self):
+        from sqlalchemy import create_engine
+        deployments = []
+        presto_engine = create_engine('presto://localhost:3894')
```

Comment:
This can be read from the suprset_config

Reply:
what would I import it as?

Reply:
Call it PRESTO_SQLALCHEMY_URI